### PR TITLE
Enlarge hitbox size of file list items

### LINKED ISSUE #1743: List item: three dots/share icon not full height
Shared icon / three dots icon in list view do not have full height, so if one is clicking nearly directly on the separation line, it will open the file/folder instead of the icon action.
(found by @jancborchardt at tablet)

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Fixes #1743 by enlarging the padding of the icons (hitbox)

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
pinging @mario @tobiasKaminsky for review. 
btw: drone will fail until the master is fixed...

**Comment by tobiasKaminsky** [comment]
👍 works fine :-)

**Comment by AndyScherzinger** [comment]
rebased, let's see if droneCI is happy now :)

**Comment by AndyScherzinger** [comment]
Ci is happy now, so pinging @mario for second review :)

File: scripts/lint/lint-results.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 DO NOT TOUCH; GENERATED BY DRONE
-      <span class="mdl-layout-title">Lint Report: 460 warnings</span>
+      <span class="mdl-layout-title">Lint Report: 447 warnings</span>
```
Comment (tobiasKaminsky) [inline scripts/lint/lint-results.txt]: Great 👍 
We slowly getting to 0 ;-)

**Comment by mario** [comment]
👍

**Pull request merged**